conf/nginx.conf

```diff
@@ -30,17 +30,62 @@ server {
 	# Add index.php to the list if you are using PHP
 	index index.html index.htm index.nginx-debian.html;
 
-	location ~* \.(json|csv)$ {
-		add_header Access-Control-Allow-Origin *;
+	# Warning: headers added by add_header are overriden by
+	# other add_header directives at the same level.
+	#location ~* \.(json|csv)$ {
+	#	add_header Access-Control-Allow-Origin *;
+	#}
+
+	location /data/ {
+     if ($request_method = 'OPTIONS') {
+        add_header Access-Control-Allow-Origin *;
+        add_header Access-Control-Allow-Methods 'GET, POST, OPTIONS';
+        #
+        # Custom headers and headers various browsers *should* be OK with but aren't
+        #
+        add_header Access-Control-Allow-Headers 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,If-None-Match';
+        #
+        # Tell client that this pre-flight info is valid for 20 days
+        #
+        add_header Access-Control-Max-Age 1728000;
+        add_header Content-Type 'text/plain; charset=utf-8';
+        add_header Content-Length 0;
+        return 204;
+     }
+     if ($request_method = 'POST') {
+        add_header Access-Control-Allow-Origin *;
+        add_header Access-Control-Allow-Methods 'GET, POST, OPTIONS';
+        add_header Access-Control-Allow-Headers 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,If-None-Match';
+        add_header Access-Control-Expose-Headers 'Content-Length,Content-Range';
+     }
+     if ($request_method = 'GET') {
+        add_header Access-Control-Allow-Origin *;
+        add_header Access-Control-Allow-Methods 'GET, POST, OPTIONS';
+        add_header Access-Control-Allow-Headers 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,If-None-Match';
+        add_header Access-Control-Expose-Headers 'Content-Length,Content-Range';
+     }
+	include snippets/ssl-headers.conf;
```

Comment:
snippets/ssl-headers.conf does does not currently exist in the Docker container

Reply:
In order to make it work, we need to update `docker-compose.yml` and add a bind to it:

```
- ./conf/nginx.conf:/etc/nginx/conf.d/default.conf
- ./conf/snippets/ssl-headers.conf:/etc/nginx/conf.d/snippets/ssl-headers.conf
```

Reply:
I just removed it.